src/commands/watch-address.cpp

```diff
@@ -68,7 +68,8 @@ console_result watch_address::invoke(std::ostream& output, std::ostream& error)
         if (!state.succeeded(ec))
             return;
 
-        state.output(format(BX_WATCH_ADDRESS_WAITING) % address);
+        if (tx_hash == null_hash)
```

Comment:
I don't follow this logic, only watch a null_hash?

Reply:
If I recall correctly, it's called multiple times and the first time it's with a null_hash, and a populated hash the other times.  Without a simple check like that, it outputs that it's watching the address multiple times.

Reply:
I suppose the first response is the result of the subscription completion.